cli/global_state.rs

```diff
@@ -118,8 +115,9 @@ impl GlobalState {
     let compile_lock = self.compile_lock.lock().await;
 
     let compiled_module = match out.media_type {
-      msg::MediaType::Unknown => state1.js_compiler.compile(out).await,
-      msg::MediaType::Json => state1.json_compiler.compile(&out).await,
+      msg::MediaType::Json | msg::MediaType::Unknown => {
```

Comment:
I think `MediaType::Json` could now be removed; it'd be handled as `MediaType::Unknown`